scripts/neutron-band-structure.py

```diff
@@ -0,0 +1,209 @@
+#! /usr/bin/env python3
+import argparse
+import os
+import pathlib
+from typing import List, Tuple
+
+import euphonic
+from euphonic import ureg
+import euphonic.plot
+import matplotlib.pyplot as plt
+import numpy as np
+import seekpath
+
+
+def get_parser() -> argparse.ArgumentParser:
+    parser = argparse.ArgumentParser()
+    parser.add_argument('file', type=str,
+                        help=('File with force constants data. Supported '
+                              'formats: .yaml (Phonopy); .castep_bin, .check '
+                              '(Castep); .json (Euphonic)'))
+    parser.add_argument('--ebins', type=int, default=200)
```

Comment:
There should be some help for `ebins`, (or maybe it should be `n-ebins`?) because from the CL initially I wasn't sure what it wanted